packages/next-server/lib/head.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types'
 import sideEffect from './side-effect'
 
 class Head extends React.Component {
+  static propTypes = {
```

Comment:
Can you assign it below the class in a `if(typeof process.env.NODE_ENV !== 'production')` so that it's removed in production, as React doesn't do type checking, you can see an example in lib/link.js

Reply:
Updated